### ISSUE TITLE
Using EJS tags in the `<title>` element will result in an error

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#1476]
- Markuplint version: `4.1.0`
- Parser lang: EJS
- Node.js version: `20.11.0`
- OS: Windows

## Describe the bug

Using EJS tags in the `<title>` element will result in an error.

## Code Example

.markuplintrc
```json
{
	"parser": {
		".ejs$": "@markuplint/ejs-parser"
	},
	"rules": {
		"character-reference": true
	}
}

```

test.ejs
```html
<!DOCTYPE html>
<html lang="en">
	<head>
		<title>title <%= title %> title</title> <!-- ← error: 不正な文字は文字参照でエスケープするべきです (character-reference) -->
	</head>
	<body>
		<h1>heading <%= heading %> heading</h1> <!-- ← no errors -->
		<p>paragraph <%= paragraph %> paragraph</p> <!-- ← no errors -->
	</body>
</html>
```

FYI, markup patterns
```html
<title><%= title %></title> <!-- ← no errors -->
<title><%= title %> title</title> <!-- ← no errors -->
<title>title <%= title %></title> <!-- ← error -->
<title>title <%= title %> title</title> <!-- ← error -->
```

FYI, version patterns

- Markuplint v3.15.0: no errors
- Markuplint v4.0.0: error
- Markuplint v4.1.0: error

## Steps To Reproduce

1. `npm i -D markuplint@4 @markuplint/ejs-parser@4`
1. `npx markuplint -c .markuplintrc test.ejs`

## The current behavior

Two errors occurred

## The expected behavior

all passed
